matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/Session.kt

```diff
@@ -294,6 +294,11 @@ interface Session :
          */
         fun onNewInvitedRoom(session: Session, roomId: String) = Unit
 
+        /**
+         * Called after sync and provide number of spaces user joined to track in analytics
+         */
+        fun onJoinedSpaceCounted(session: Session, numSpaces: Int) = Unit
```

Comment:
Have you considered implementing that in the interface `StatisticEvent` instead? I think it will have more sense.

Reply:
Yes, I was thinking about it, but usage of `StatisticsEvent` is

    ```
override fun onStatisticsEvent(session: Session, statisticEvent: StatisticEvent) {
        statisticEvent.toListOfPerformanceTimer().forEach {
            analyticsTracker.capture(it)
        }
    }
```
So I'll either need to add some ugly workaround for a new Event to not resolve one specific case to `PerfomanceTimer` Event, or rework it completely. Wasn't sure that this change worth it. What do you think?

Reply:
Yes, I have seen that, I guess we will have to update the code on the app side (change `statisticEvent.toListOfPerformanceTimer()`). But I think we will end up with a better SDK API, which is a higher priority. Can you do it please?